velox/common/base/tests/CMakeLists.txt

```diff
@@ -50,7 +50,8 @@ target_link_libraries(
           gflags::gflags
           gtest
           gmock
-          gtest_main)
+          gtest_main
+          fast_float)
```

Comment:
Is this change needed?

Reply:
Yes, Without this line, the following exception will occur when compiling velox:
```
FAILED: velox/common/base/tests/CMakeFiles/velox_base_test.dir/SpillConfigTest.cpp.o
/opt/homebrew/bin/ccache /Library/Developer/CommandLineTools/usr/bin/c++ -DBOOST_ATOMIC_DYN_LINK -DBOOST_ATOMIC_NO_LIB -DBOOST_CONTEXT_DYN_LINK -DBOOST_CONTEXT_NO_LIB -DBOOST_FILESYSTEM_DYN_LINK -DBOOST_FILESYSTEM_NO_LIB -DBOOST_PROGRAM_OPTIONS_DYN_LINK -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_REGEX_DYN_LINK -DBOOST_REGEX_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_DYN_LINK -DBOOST_THREAD_NO_LIB -DFOLLY_HAVE_INT128_T=1 -DGFLAGS_IS_A_DLL=0 -I/Users/wypb/data/code/apache/velox/_build/release/_deps/protobuf-src/src -I/Users/wypb/data/code/apache/velox/. -I/Users/wypb/data/code/apache/velox/velox/external/xxhash -I/Users/wypb/data/code/apache/velox/_build/release/_deps/xsimd-src/include -isystem /Users/wypb/data/code/apache/velox/velox -isystem /Users/wypb/data/code/apache/velox/velox/external -isystem /opt/homebrew/include -isystem /usr/local/include -isystem /opt/homebrew/Cellar/openssl@1.1/1.1.1w/include -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src/googletest/include -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src/googletest -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src/googlemock/include -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src/googlemock -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src/include -isystem /Users/wypb/data/code/apache/velox/_build/release/_deps/gtest-src -mcpu=apple-m1+crc -std=c++17 -fvisibility=hidden -D USE_VELOX_COMMON_BASE -D HAS_UNCAUGHT_EXCEPTIONS -Wall -Wextra -Wno-unused        -Wno-unused-parameter        -Wno-sign-compare        -Wno-ignored-qualifiers        -Wno-range-loop-analysis          -Wno-mismatched-tags          -Wno-nullability-completeness -O3 -DNDEBUG -std=gnu++17 -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk -fPIE -fcolor-diagnostics -fcolor-diagnostics -MD -MT velox/common/base/tests/CMakeFiles/velox_base_test.dir/SpillConfigTest.cpp.o -MF velox/common/base/tests/CMakeFiles/velox_base_test.dir/SpillConfigTest.cpp.o.d -o velox/common/base/tests/CMakeFiles/velox_base_test.dir/SpillConfigTest.cpp.o -c /Users/wypb/data/code/apache/velox/velox/common/base/tests/SpillConfigTest.cpp
In file included from /Users/wypb/data/code/apache/velox/velox/common/base/tests/SpillConfigTest.cpp:20:
In file included from /Users/wypb/data/code/apache/velox/./velox/exec/HashBitRange.h:18:
In file included from /Users/wypb/data/code/apache/velox/./velox/core/PlanNode.h:18:
In file included from /Users/wypb/data/code/apache/velox/./velox/connectors/Connector.h:25:
In file included from /Users/wypb/data/code/apache/velox/./velox/core/ExpressionEvaluator.h:18:
In file included from /Users/wypb/data/code/apache/velox/./velox/vector/ComplexVector.h:26:
In file included from /Users/wypb/data/code/apache/velox/./velox/vector/BaseVector.h:35:
In file included from /Users/wypb/data/code/apache/velox/./velox/type/Variant.h:26:
In file included from /Users/wypb/data/code/apache/velox/./velox/type/Conversions.h:24:
/Users/wypb/data/code/apache/velox/./velox/type/StringToFloatParser.h:19:10: fatal error: 'fast_float/fast_float.h' file not found
#include <fast_float/fast_float.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
> /Users/wypb/data/code/apache/velox/./velox/type/StringToFloatParser.h:19:10: fatal error: 'fast_float/fast_float.h' file not found
#include <fast_float/fast_float.h>

I assume this include can be removed, no?

Reply:
`fast_float::from_chars` is used in `velox/type/StringToFloatParser.cpp`

Reply:
I should understand what you mean. I should move `#include <fast_float/fast_float.h>` to `velox/type/StringToFloatParser.cpp` so I can delete this line.